src/plugins/elasticsearch/index.js

```diff
@@ -23,7 +23,8 @@ module.exports = function (kibana) {
           cert: Joi.string(),
           key: Joi.string()
         }).default(),
-        minimumVerison: Joi.string().default('1.4.4')
+        apiVersion: Joi.string().default('master'),
```

Comment:
is it a good idea to peg the client version to master? It was on 1.4 earlier, maybe we should pick a version?